client/lobby/RandomMapTab.cpp

```diff
@@ -107,7 +107,7 @@ RandomMapTab::RandomMapTab():
 	//new callbacks available only from mod
 	addCallback("teamAlignments", [&](int)
 	{
-		GH.windows().createAndPushWindow<TeamAlignmentsWidget>(*this);
+		GH.windows().createAndPushWindow<TeamAlignments>(*this);
```

Comment:
Why we now need two windows for this?
TeamAlignmentsWidget and TeamAlignments?
What's the difference between them?

Reply:
One is the Window, the other a Widget inside of this Window. I don't find a nice way to add border without it.